bootstrap/config/configuration.go

```diff
@@ -12,7 +12,7 @@
  * the License.
  *******************************************************************************/
 
-package configuration
+package config
```

Comment:
Package name is `config`; usage is typically `config.[function]`.  For readability's sake, update package's function names to drop "config" from name; e.g. `config.UpdateFromConfigProvider` -> `config.UpdateFromProvider`.

Reply:
There are remnants; config.ConfigRegistryStemCore -> config.RegistryStemCore; config.ConfigMajorVersion -> config.MajorVersion; etc. 

Newly created provider.go should reflect this also; config.ConfigProviderInfo -> config.ProviderInfo; config.NewConfigProviderInfo -> config.NewProviderInfo; etc.